astropy/table/tests/conftest.py

```diff
@@ -137,13 +137,22 @@ def table_type(request):
               'time': time.Time([2000, 2001, 2002, 2003], format='jyear'),
               'skycoord': coordinates.SkyCoord(ra=[0, 1, 2, 3] * u.deg,
                                                dec=[0, 1, 2, 3] * u.deg),
+              'sphericalrep': coordinates.SphericalRepresentation(
```

Comment:
Is this enough to test that roundtrip through FITS or ECSV works?  I forget...

Reply:
I fear you are right that it is not...

Reply:
now fixed